docs/commands/move.md

```diff
@@ -0,0 +1,33 @@
+# `move` command
+
+Note: The implementation for `copy` and `move` are exactly the same. The only different is that move will remove the source after a successful copy.
+
+The `move` commands works like the `mv` or `rsync` binaries. It allows moving either single entries or whole folders around. Moving across mounts is supported.
+
+If the source is a directory the source directory is re-created at the destination if no trailing slash is found. Otherwise the contained secrets are flattened (similar to what `rsync` does).
+
+Please note that `move` will always decrypt the source and re-encrypt at the destination.
+
+## Synopsis
+
+```
+$ gopass move path/to/leaf new/leaf
+$ gopass move path/to/dir new/dir
+```
+
+## Modes of operation
+
+* Move a single secret from source to destination
+* Move a folder of secrets, possibly with sub folders, from source to destination
+
+## Flags
+
+Flag | Aliases | Description
+---- | ------- | -----------
+`--force` | `-f` | Overwrite existing destination without asking.
+
+## Details
+
+* To simplify the implementation and support multiple backends a `copy` or `move` operation will always decrypt and re-encrypt all affected secrets. Even if moving encrypted files around might be possible.
+* You can move a secret to another secret, i.e. overwrite the destination. But `gopass` won't let you move a directory over a file. In that case you have to delete the destination first.
+
```

Comment:
What should move do if we move a secret onto itself? 

For instance would `gopass move /entry /entry` rencrypt the secret or do nothing?